### ISSUE TITLE
(node:10800) UnhandledPromiseRejectionWarning when launching with headless: false & slowMo ON

### ISSUE THREAD

**Issue by Tynael** [puppeteer/puppeteer#616]
I've come across an unexpected error when launching Puppeteer with `headless: false` and `slowMo: 250`.

I'm running on Windows 10 with Node 8.4.0 installed.

This is the error:

`(node:10800) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Erk.continueInterceptedRequest): Target closed.
(node:10800) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the that are not handled will terminate the Node.js process with a non-zero exit code.
(node:10800) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): AN]: false == true`

And here is the part of the code causing this trouble:

```
'use strict';

const puppeteer = require('puppeteer');

(async() => {

  const browser = await puppeteer.launch(
    {
      headless: false,
      slowMo: 250
    }
  );
const page = await browser.newPage();
await page.setRequestInterceptionEnabled(true);
  page.on('request', request => {
    if (request.url) {
        var networkRequests = request.url;
        var raRequest = JSON.stringify(networkRequests);
        console.log(raRequest);
        console.log(typeof raRequest);
    } else {
        request.abort();
    }
    request.continue();
  });
await page.goto('https://example.com');


browser.close();

})();
```

Hope it helps.

P.S: If something is wrong with that code, please do let me know!

**Comment by JustinYi922** [comment]
升级一下linux的nss。我昨晚试了，pupeteer可以工作了。

**Comment by Tynael** [comment]
@JustinYi922 I've Google Translate-ed your comment as I don't understand Chinese :D and from what I understood, you told me to update my Linux OS. I'm running on Windows 10. I've updated the main post accordingly.

**Comment by aslushnikov** [comment]
@Tynael if you add the `await page.close()` right before the `browser.close()`, the error should disappear.

This, however, is a problematic space for puppeteer: browser closing races with request interception, and this is a very common scenario. This should be fixed in puppeteer.

**Comment by Tynael** [comment]
I understand. Thanks for the clarification @aslushnikov .

**Comment by aslushnikov** [comment]
This should be fixed with 0bea42bd8c4002ed28d100d9ccc06c6ac03a9b70
